app/services/WhatDoYouWantToDoRadioToggleService.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import config.FrontendAppConfig
+import connectors.BetaAuthorizationConnector
+import logging.Logging
+import models.EoriNumber
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class WhatDoYouWantToDoRadioToggleService @Inject()(
```

Comment:
This seems to be a duplication of `DisplayDeparturesService`. We can reuse that instead of a new class

Reply:
There's a slight difference between the 2 services, the new one we've created doesn't utilise the departureJourney toggle and it's primarily used for redirecting to the old service when departures is disabled

Reply:
After talking about this, we're agreed that the subtle difference of the two means warrants a different flag checker